tests/helpers.bash

```diff
@@ -139,6 +139,22 @@ function wait_for_policy_enforcement {
     done
 }
 
+function wait_for_cilium_k8s_ep_gen {
+    export cilium_k8s_npods=$(kubectl get pods -n kube-system | grep cilium | wc -l)
+    set +x
+    { for i in $(seq 1 ${cilium_k8s_npods}); do
+        while true; do
+            cilium_pod_id=$(kubectl get pods -n kube-system | grep cilium | awk "NR==$i{ print \$1 }")
+            if ! kubectl exec -n kube-system -i ${cilium_pod_id} cilium endpoint list | grep regenerating; then
```

Comment:
I've added a FIXME and added a FIXME on the function above since it might have the same problem.